NewForce-Cohort-5/nutshell-el-toro-loco
# Jt message form

# Description
I have completed the message form and list elements, so that it prints with users email address prepended to the messages sent. The chat box's size will remain the same and  a scroll bar will be added when necessary. I have also added data provider for users API with appropriate fetch calls. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
-you will need to make sure you json-server is running
-log into the app with an existing users email or register for new
-type message into chat input and press send to see message posted to chat-list with the newest chat at the bottom
-press modify button and the message should repopulate into the input to be edited and replaced
-save newly edited chat message
-still working on cancel
# Checklist:
